src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.tsx

```diff
@@ -175,7 +231,20 @@ export const TranscriptsListItemInfo = (
           </div>
         </div>
 
-        <div className={styles.moreInformation}>More information</div>
+        {!!(transcriptMetaData || transcriptCCDS) && (
```

Comment:
Is it better to make these variables boolean to avoid `!!` everywhere?

Reply:
I am using transcriptCCDS to display data. 

```
          <div className={styles.moreInfoColumn}>
            <ExternalReference
              label={'CCDS'}
              to={transcriptCCDS.url}
              linkText={transcriptCCDS.accession_id}
            />
```